fabric-world-v1/build.gradle

```diff
@@ -0,0 +1,6 @@
+archivesBaseName = "fabric-world-v1"
```

Comment:
Would this fit better in `fabric-object-builders` im not sure it needs a whole new module

Reply:
:+1: This is really just a convenient alternative builder for the chunk generator object; it indeed does not mingle with worldgen logic.